### ISSUE TITLE
111.28.00 crashes on initialization

### ISSUE THREAD

**Issue by xandkar** [janestreet/async_extended#1]
I'm on Mac OS X 10.9.4. The minimal case is bellow:

```
$ opam info -f version async_extended
111.28.00

$ cat foo.ml
module Ext = Async_extended.Std

$ ocamlfind ocamlc -g -linkpkg -package async_extended -thread foo.ml -o foo

$ ./foo
Fatal error: exception Info.Exn(_)
Raised at file "lib/error.ml", line 7, characters 21-29
Called from file "lib/deprecated_async_bench.ml", line 200, characters 20-47

```

**Comment by xandkar** [comment]
With 112.01.00 I get a more informative exception, originating in the same spot:

```
$ opam info -f version async_extended
112.01.00

$ cat foo.ml
module Ext = Async_extended.Std

$ ocamlfind ocamlc -g -linkpkg -package async_extended -thread foo.ml -o foo

$ ./foo
Fatal error: exception (unimplemented Posix_clock.gettime)
Raised at file "lib/error.ml", line 7, characters 21-29
Called from file "lib/deprecated_async_bench.ml", line 200, characters 20-47
```

**Comment by agarwal** [comment]
```
Core_extended.Posix_clock.gettime;;
- : (Core_extended.Posix_clock.t -> Int63.t) Or_error.t =
Core_kernel.Result.Error (unimplemented Posix_clock.gettime)
```

This function is indeed not available on Mac OS, and this does seem like a bug. `Deprecated_async_bench` defines the value `bench_raw`, which calls `Ok_error.ok_exn bench_basic`, which calls `Posix_clock.gettime`.

**Comment by smolkaj** [comment]
Any updates on this?

**Comment by bmillwood** [comment]
Sorry this took so long to reply to.

I think I'll just delete the entire module. It's deprecated and (as far as I'm aware) unused.

**Comment by bmillwood** [comment]
This is done internally. Should be included in any version ≥ 113.33.

I didn't actually test that it fixes initialisation on MacOS. Feel free to re-open this issue if there's more to do.
